CHANGELOG.md

```diff
@@ -5,9 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog][chg] and this project adheres to
 [Haskell's Package Versioning Policy][pvp]
 
-## unreleased - 2023-07-06
+## unreleased - 2023-11-23
 
   - Add support for aeson 2.2
+  - `fallibleRuntime` and `fallibleRuntimeWithContext` reports errors to AWS
+    instead of returning `{"Left": "some error"}`.
```

Comment:
Probably worth noting that the `Either` is totally unwrapped, successful responses are no longer wrapped in `{ "Right": ... }` also.

Reply:
Added another line describing what happens for the successful responses.